packages/components/src/components/event/eventHero/EventHero.tsx

```diff
@@ -32,21 +16,42 @@ import {
   PageSection,
   useConfig,
 } from 'react-helsinki-headless-cms';
-import AppConfig from '../../app/AppConfig';
-
-import EventKeywords from '../eventKeywords/EventKeywords';
+import type {
+  AppConfig,
+  EventFields,
+  ExtractLatestReturnPathFunction,
+  ExtractLatestReturnPathReturnParams,
+  SuperEventResponse,
+} from '../../../index';
+import {
+  useLocale,
+  getDateRangeStr,
+  buttonStyles,
+  IconButton,
+  InfoWithIcon,
+  SkeletonLoader,
+  EventLocationText as LocationText,
+  getEventFields,
+  getEventPrice,
+  EventKeywords,
+} from '../../../index';
 import EventName from '../eventName/EventName';
-import type { ReturnParams } from '../eventQueryString.util';
-import { extractLatestReturnPath } from '../eventQueryString.util';
 import styles from './eventHero.module.scss';
 
-export interface Props {
+export interface EventHeroProps {
   event: EventFields;
   superEvent?: SuperEventResponse;
+  appConfig: AppConfig;
```

Comment:
Instead of giving the whole AppConfig, only the configurations that we need should be given.